torch/csrc/utils/init.cpp

```diff
@@ -2,6 +2,8 @@
 #include <torch/csrc/utils/init.h>
 #include <torch/csrc/utils/throughput_benchmark.h>
 
+#include <ATen/native/Convolution.h>
```

Comment:
Could you add a comment here to say this is a temporary workaround? (Or is there is better place to add the binding? because technically jit shouldn't need to depend on conv).

Reply:
This isn't in a jit folder though?